docs/assets/css/secondary-nav.less

```diff
@@ -1,85 +1,178 @@
+                margin-top: unit( 5px / @base-font-size-px, em );
         });
 
-        &__current {
-            .respond-to-max(800px, {
-                display: none;
+        .m-list_item {
+            .respond-to-min(901px, {
+                margin-bottom: unit( 4px / @base-font-size-px, em );
+            });
+        }
+
+        .m-list_link {
+            padding-left: unit( 20px / @base-font-size-px, em );
+            .respond-to-max(900px, {
+                border-style: solid;
+                border-color: @gray-40;
             });
-            .respond-to-min(801px, {
-                .u-link__colors(@black, @black, @black, @black, @black,
-                                @green, @green, @green, @green, @green);
+            .respond-to-min(@bp-sm-min, {
+                margin-left: unit( 20px / @base-font-size-px, em );
+                padding: unit( 4px / @base-font-size-px, em ) unit( 15px / @base-font-size-px, em );
+                border-bottom: none;
+                color: @black;
+                &:hover,
+                &:focus {
+                    background: @gray-10;
+                }
             });
         }
+    }
+}
+
+
+//
+// Mixin: Bleedbar sidebar (on left) styling
+// @breakpoint argument: pixel value of minimum screen width at which you want the bleedbar styling to appear
+//
+
+.content__bleedbar_left {
+    // Overrides the border between main and sidebar, because bleedbar makes its own.
+    .content_main:after {
+        content: none;
+    }
+
+    .respond-to-min( @bp-med-min, {
+        // Prevent pseudo element background from creating horizontal scrollbar.
+        overflow: hidden;
+
+        .content_main {
+            padding-top: unit( ( @grid_gutter-width * 1.5 ) / @base-font-size-px, em );
+            padding-right: 0;
+            padding-bottom: unit( ( @grid_gutter-width / 2) / @base-font-size-px, em );
+            padding-left: unit( @grid_gutter-width / @base-font-size-px, em );
+            margin-left: 0;
+            position: relative;
+            z-index: 1;
+
+            // Remove background color from actual sidebar.
+            // It will now be provided by pseudoelement :after sidebar parent.
+            background: transparent;
+        }
 
-        &__button {
-            width: 100%;
-            font-weight: bold;
-            text-align: left;
-            text-transform: uppercase;
+        .content_wrapper {
+            position: relative;
 
-            &:focus {
-                outline: thin dotted;
+            &:before {
+                // Create faux sidebar background.
+                content: '';
+                display: block;
+                width: 9999px;
+                border-right: 1px solid @content_sidebar-border;
+                height: 100%;
+                position: absolute;
+                top: 0;
+                z-index: 0;
+                // The three in the below calculation is a magic number.
```

Comment:
Will 3 always work if the `@grid_gutter-width` value were to change? Or is perhaps `10px` the magic number and that should be what is below?